src/pathfinding_service/model/token_network.py

```diff
@@ -140,6 +141,32 @@ def _calculate_fees(self) -> Optional[List[FeeAmount]]:
 
         return fees
 
+    def _get_matrix_users(self) -> Optional[Dict[str, str]]:
+        # Check node reachabilities
+        user_ids: Dict[str, str] = {}
+        for node in self.nodes:
+            node_user_ids = self.reachability_state.get_userids_for_address(node)
+            node_user_id = None
+            for user_id in node_user_ids:
+                if self.reachability_state.get_userid_presence(user_id) in [
+                    UserPresence.ONLINE,
+                    UserPresence.UNAVAILABLE,
+                ]:
+                    node_user_id = user_id
+                    # if a reachable user is found we arbitrarily choose
+                    # this user for the given address. There should not be another user online
+                    break
+
+            if node_user_id is None:
```

Comment:
This is always true when this code is reached, isn't it?

If this `if` can be removed, then line 149 is also unnecessary.

Reply:
In line 155 `node_user_id` is assigned if the user_id is online

Reply:
I've seen that, but I paired the `break` with the wrong `for` while writing that comment.